modules/installation-aws-outposts-supported-instance-types.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
```

Comment:
Suggest updating the file name to `installation-identify-supported-instance-types`

Reply:
File renamed as suggested